src/core/lib/iomgr/resolve_address_custom.cc

```diff
@@ -88,15 +88,21 @@ void grpc_custom_resolve_callback(grpc_custom_resolver* r,
 static grpc_error* try_split_host_port(const char* name,
                                        const char* default_port, char** host,
                                        char** port) {
-  /* parse name, splitting it into host and port parts */
   grpc_error* error;
-  gpr_split_host_port(name, host, port);
-  if (*host == nullptr) {
-    char* msg;
-    gpr_asprintf(&msg, "unparseable host:port: '%s'", name);
-    error = GRPC_ERROR_CREATE_FROM_COPIED_STRING(msg);
-    gpr_free(msg);
-    return error;
+  *port = nullptr;
+  // all hosts beginning with / are treated as unix domain sockets
+  if (name[0] == '/') {
```

Comment:
The logic for determining whether a name is a UDS name and for handling those names should be the same as in the other resolver implementation(s). See https://github.com/grpc/grpc/blob/master/src/core/lib/iomgr/resolve_address_posix.cc#L57.

Reply:
fixed. thanks 👍